README.md

```diff
@@ -14,30 +14,76 @@ This README contains a high level description of contributing a SWIP, for conven
-is no rough consensus - for instance, because contributors point out significant issues with the SWIP - they may close the PR and 
-request that you fix the issues in the draft before trying again.
+### Creating a new SWIP
 
-For a detailed description of the SWIP process see [SWIP-0](SWIPs/swip-0.md).
+A new SWIP can be proposed by opening a PR in the [SWIP repo](https://github.com/ethersphere/SWIPs) titled `SWIP-<XXXX>: <title>` where `XXXX` is the SWIP number left padded with zeros up to four digits (i.e., fprinted with the format string `%04d`). 
+The PR is adding at least a new placeholder file for the new proposal named `XXXX.md` to be located in the `SWIPs` subdirectory of the SWIPs repo.
 
-# SWIP Status Terms
+This placeholder file contains minimally only the the yaml header with the following fields:
 
-* **Draft** - a SWIP that is undergoing rapid iteration and changes.
-* **Last Call** - a SWIP that is done with its initial iteration and ready for review by a wide audience.
-* **Accepted** - a core SWIP that has been in Last Call for at least 2 weeks and any technical changes that were requested have been addressed by the author. The process for Core Devs to decide whether to encode a SWIP into their clients as part of a mandatory update is not part of the SWIP process. If such a decision is made, the SWIP will move to final.
-* **Final (non-Core)** - a SWIP that has been in Last Call for at least 2 weeks and any technical changes that were requested have been addressed by the author.
-* **Final (Core)** - a SWIP that the Core Devs have decided to implement and release in a future mandatory update or has already been released as such. 
-* **Deferred** - a SWIP that is not being considered for immediate adoption. May be reconsidered in the future.
+```yaml
+ ---
+title: fairx
+SWIP: XXXX
+author: Viktor Tron (@zelig), Viktor Toth (@nugaon)
+status: Draft
+---
+```
+
+- The title must be descriptive yet concise
+- SWIP is the SWIP number and must be the same as the filename
+- Authors must be specified as a comma-separated list of `real name (@github_handle)`
+- The status must be initially set to `Draft`. see below the section on status.
+
+The SWIP editors can merge this initial request to signal the SWIP worthy of recording and ready for discussion.
+
+From this moment on, the SWIP number is taken as allocated and the SWIP will permanently be part of the repo (yes even if it is deferred or rejected).
+This way looking at the repo should give the number to be used when  a new SWIP proposal is submitted. 
+
+Submitters of SWIPs are recommended to use a mnemonic feature branch name. 
+
+### Updating a SWIP
+
+A suggested edit to a SWIP can be proposed by opening a PR in the [SWIP repo](https://github.com/ethersphere/SWIPs) with a PR title in the following format: `Update SWIP-XXXX: Title of the suggested change`. 
```

Comment:
This is for an edit of a SWIP that is already final / accepted? As a draft SWIP should be just edited directly?

Reply:
Actually, reading forward, it is for a SWIP that is merged but can be in any state?